### PR TITLE
Added first draft of helper methods

### PR DESCRIPTION
**Pull request by garethahealy**
#### What is this PR About?
$subject

#### How do we test this?
Action is green

cc: @redhat-cop/bats-library

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Testing
+Each BATS addon must be tested by its own [BATS test](https://github.com/garethahealy/bats-library/blob/master/test/tests.sh#L5-L12).
```
Comment (springdo) [inline TESTING.md:2]: probs a broken url here right?

File: test/data/json-root.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "kind": "Tesy McTester",
```
Comment (springdo) [inline test/data/json-root.json:2]: 💯 for the names here

File: TESTING.md (hunk 1 above)
Comment (garethahealy) [reply TESTING.md]: fixed to point to redhat-cop - but ye, its a deadlink until merged

File: test/tests.sh [hunk 3]
```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bats
+
+load test_helper/bats-support/load
```
Comment (springdo) [inline test/tests.sh:3]: Wonder if we could change this to bring the dep in by cloning first - then we would not need to check in all of `bats-support` to the test folder

**Comment by springdo** [comment]
@garethahealy I'll merge this in for now and we can circle back to the `git clone` or even wget the `bats-support` dep from github before loading it

**Pull request merged**